test/testcommon/src/main/java/io/pravega/test/common/SerializedClassRunner.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.common;
+
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runners.BlockJUnit4ClassRunner;
+import org.junit.runners.model.InitializationError;
+
+public class SerializedClassRunner extends BlockJUnit4ClassRunner {
```

Comment:
For completeness, why did you have to write this class and what's that `RunNotifier`?

Reply:
Added a Javadoc.
It's needed for some metrics tests which cannot be run in parallel.
RunNotifier is a Junit class. It's not really being used here except as a pass-through to the parent implementation.